tests/ui/simd/intrinsic/float-minmax-pass.rs

```diff
@@ -7,11 +7,22 @@
 #![allow(non_camel_case_types)]
 
 #[repr(simd)]
-#[derive(Copy, Clone, PartialEq, Debug)]
+#[derive(Copy, Clone)]
 struct f32x4(pub [f32; 4]);
+impl f32x4 {
+    fn to_array(self) -> [f32; 4] { unsafe { std::mem::transmute(self) } }
+}
 
 use std::intrinsics::simd::*;
 
+macro_rules! all_eq {
+    ($a: expr, $b: expr) => {{
+        let a = $a;
+        let b = $b;
+        assert_eq!(a.to_array(), b.to_array());
```

Comment:
is this necessary? can we not just use `$a.to_array()`? Also maybe rename this to `assert_eq_simd!` like the test below?